resources/plugins/arty/CG_ArtySpotter_V1_2a_MP.lua

```diff
@@ -0,0 +1,715 @@
+-- Artillery Spotter script - Multiplayer version
+-- by Carsten Gurk aka Don Rudi
+
+local version = "MP 1.2"
+
+-- User configurable variables
+
+local user_fireDelay = 10				-- time to impcat of the rounds
+local user_quantity = 20				-- how many rounds will be fired in a fire for effect task
+local user_spread = 50					-- impact radius of the rounds during fire for effect
+local user_spottingDistance = 15		-- max allowable distance from player to target to prevent cheating. In kilometers.
+local user_restrictByType = ""        	-- Restriction by type ("", "helo", etc.)
+local user_restrictByUnitName = "" 		-- Restriction by unit name ("", "spotter", etc.), not case sensitive	
```

Comment:
We should probably refactor this so that these settings can be set through Retribution. I can handle that part if you're unfamiliar with the technicalities... 😅